client/js/screens/Profile/styles.js

```diff
@@ -0,0 +1,55 @@
+import theme from "../../config/globalStyles";
+
+const styles = {
+  container: {
+    ...theme.container,
+    height: "100%",
+    marginLeft: "7%",
+    marginRight: "7%",
```

Comment:
Please use `marginHorizontal: (int value)`. Thank you.

Reply:
Done.